### PR TITLE
Bug 1926285: UPSTREAM: <carry>: ignore not found errors in status messages

### PR DESCRIPTION
**Pull request by rphillips**
Ignore not found errors within status messages.

### CODE REVIEW

**Comment by deads2k** [comment]
This lgtm, but I'll let someone from the node or networking team have a look too.  This fixes the event stream for any pods, not just platform pods, so I think it's valid to backport too.

/lgtm
/hold

holding to confirm it fixes the issue and in case you'd like someone else to have a look.

**Review by harche — APPROVED** [review]
/lgtm

File: pkg/kubelet/kuberuntime/kuberuntime_manager.go [hunk 1]
```diff
@@ -72,6 +73,10 @@ const (
 var (
 	// ErrVersionNotSupported is returned when the api version of runtime interface is not supported
 	ErrVersionNotSupported = errors.New("runtime api version is not supported")
+	// ignore error regexex for sandbox creation
+	ignoreSandboxCreateErrorRegex = []*regexp.Regexp{
```
Comment (smarterclayton) [inline pkg/kubelet/kuberuntime/kuberuntime_manager.go:77]: As a follow up can we make the CRI and container runtimes return structured errors?  @mrunalp and I had discussed it, but I didn't know whether it landed (GRPC gives us the tools to have the runtimes do the right thing and give us a typed error).

Comment (rphillips) [reply pkg/kubelet/kuberuntime/kuberuntime_manager.go:77]: Yes. +1

**Pull request merged**